pkgs/tools/misc/lottieconverter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, rlottie, libpng, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "LottieConverter";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "sot-tech";
+    repo = pname;
+    rev = "r${version}";
+    hash = "sha256:11dribhqbb6rdrbr86a9sk9iwyv6097xazp31srwv3knl23v60cl";
+  };
+
+  nativeBuildInputs = [ rlottie libpng zlib ];
+
+  buildPhase = ''
+    make CONF=Release
```

Comment:
When overriding `BuildPhase`, you should also provide `runHook preBuild` and `runHook postBuild`. Though, in this case, it is better to rely on the default `buildPhase` and just set `makeFlags = [ "CONF=Release" ];`.